test/cpp/jit/test_gpu.cpp

```diff
@@ -11771,6 +11771,142 @@ TEST(NVFuserTest, FusionKirScoping_CUDA) {
   TORCH_CHECK(top_level_scope == nullptr);
 }
 
+TEST(NVFuserTest, FusionOmitPredicate1_CUDA) {
```

Comment:
I understand this is performance not correctness issue, is that right?
Shouldn't we state it as such?

Reply:
That is true. I added a comment to where it is actually done. 4aa636db79af1092726c68038df109dec1a17ec6